dist/src/main/resources/rules/file-mappings.windup.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="XmlFileMappings">
+    <rules>
+        <rule>
```

Comment:
Well, yes, but I am following the Java class structure, in which they are necessary. I would rather have only one option of these configuration rules. Otherwise it may lead to a mess.